libc-test/build.rs

```diff
@@ -2657,7 +2658,7 @@ fn test_linux(target: &str) {
         (struct_ == "timex" && field.starts_with("__unused")) ||
         // FIXME: It now takes mode_t since glibc 2.31 on some targets.
         (struct_ == "ipc_perm" && field == "mode"
-            && ((x86_64 || i686 || arm) && gnu || x86_64_gnux32)
+            && ((x86_64 || i686 || arm || riscv64) && gnu || x86_64_gnux32)
```

Comment:
ipc_perm now uses mode_t everywhere.

Reply:
Skipping this because it was introduced in the newest version of glibc and I'd like to wait for a while. On riscv, isn't it newer change?